package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "appium-mac2-driver",
+  "description": "XCTest-based Appium driver for macOS apps automation",
+  "keywords": [
+    "appium",
+    "mac",
+    "XCTest"
+  ],
+  "version": "0.0.1",
+  "author": "appium",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/appium/appium-mac2-driver.git"
+  },
+  "bugs": {
+    "url": "https://github.com/appium/appium-mac2-driver/issues"
+  },
+  "engines": [
+    "node"
+  ],
+  "appium": {
+    "driverName": "mac2",
```

Comment:
is there something more descriptive than `mac2`? is `mac` already taken by the other driver? if so, maybe we change the name of that one instead?

Reply:
yes, there is already a mac driver: https://github.com/appium/appium-mac-driver